bika/lims/exportimport/instruments/resultsimport.py

```diff
@@ -255,6 +255,70 @@ def _parseline(self, line):
         raise NotImplementedError
 
 
+class InstrumentTXTResultsFileParser(InstrumentResultsFileParser):
+
+    def __init__(self, infile, separator, encoding=None,):
+        InstrumentResultsFileParser.__init__(self, infile, 'TXT')
+        # Some Instruments can generate files with different encodings, so we
+        # may need this parameter
+        self._separator = separator
+        self._encoding = encoding
+
+    def parse(self):
+        infile = self.getInputFile()
+        self.log("Parsing file ${file_name}", mapping={"file_name":infile.filename})
+        jump = 0
+        # We test in import functions if the file was uploaded
+        try:
+            if self._encoding:
+                import codecs
+                f = codecs.open(infile.name, 'r', encoding=self._encoding)
+            else:
+                f = open(infile.name, 'rU')
+        except AttributeError:
+            f = infile
+        for line in f.readlines():
+            self._numline += 1
```

Comment:
Why is this needed?

Reply:
@ramonski As far as I checked `self._numline` is used to show the line number in the logs